ios/FluentUI/Vnext/Design Token System/ListTokens.swift

```diff
@@ -6,13 +6,6 @@
 import UIKit
 import SwiftUI
 
-/// Pre-defined styles of icons
-@objc public enum MSFListIconStyle: Int, CaseIterable {
```

Comment:
Can we incorporate the different icon sizes in this enum and bring it back?
Then we can tackle the 16x16 icon issue too.